examples/README.md

```diff
@@ -45,29 +45,59 @@ npm install @gomomento/sdk
 ### Usage
 
 ```typescript
-import {SimpleCacheClient, CacheGetStatus} from '@gomomento/sdk';
-
-// your authentication token for momento
-const authToken = process.env.MOMENTO_AUTH_TOKEN;
-
-// initializing momento
-const DEFAULT_TTL = 60; // 60 seconds for default ttl
-const momento = new SimpleCacheClient(authToken, DEFAULT_TTL);
-
-// creating a cache named "myCache"
-const CACHE_NAME = 'myCache';
-await momento.createCache(CACHE_NAME);
-
-// sets key with default ttl
-await momento.set(CACHE_NAME, 'key', 'value');
-const res = await momento.get(CACHE_NAME, 'key');
-console.log('result: ', res.text());
-
-// sets key with ttl of 5 seconds
-await momento.set(CACHE_NAME, 'key2', 'value2', 5);
-
-// permanently deletes cache
-await momento.deleteCache(CACHE_NAME);
+import {
```

Comment:
Thanks for updating this.  We should definitely merge these changes.  but...

This approach has that issue where if the API changes we won't necessarily know that the README code is broken.  I know we solved this for the main READMEs via the template.  I don't remember what we did in the other SDKs to deal with this issue in the example READMEs?

Reply:
We don't have a standard approach. The .NET example README only contains small code snippets and the PHP version points to the various examples without including any code at all. I'll make a ticket for defining and implementing a standard we can apply across SDKs.

Reply:
https://github.com/momentohq/dev-eco-issue-tracker/issues/169